composer.json

```diff
@@ -43,7 +47,6 @@
     },
     "bin": [
         "bin/expressive",
-        "bin/expressive-create-middleware",
```

Comment:
Never before released; that feature was for the upcoming 0.4.0 release. As such, I'd rather _not_ release that script, and instead ship it only as a command in the `expressive` script.